docker/scripts/release.sh

```diff
@@ -33,8 +33,8 @@ if [ "$FOUND" -ne "0" ]; then
   exit 3
 fi
 
-if [[ ! "$RELEASE_TAG" =~ ^([0-9]*\.(([0-9]*\.)*[0-9]*))$ ]]; then
-  echo "ERROR: Invalid tag. Use pattern x.y[.z]"
+if [[ ! "$RELEASE_TAG" =~ ^([0-9]+\.[0-9]+\.[0-9]+)$ ]]; then
```

Comment:
Are the parenthesis needed? Wouldn't this work too, since you're not using the capture group anywhere?
```suggestion
if [[ ! "$RELEASE_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

```
RELEASE_TAG='1.2.3'; [[ ! "$RELEASE_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]] && echo 'release tag not ok' || echo 'tag ok'
tag ok
```